index.d.ts

```diff
@@ -88,7 +88,7 @@ export interface FastifyStaticOptions extends SendOptions {
   etag?: boolean;
   extensions?: string[];
   immutable?: boolean;
-  index?: string[] | false;
+  index?: string[] | string | false;
```

Comment:
could you please update the tests for the types? We use tsd.

Reply:
@mcollina does e914e64 work?